action.yml

```diff
@@ -123,7 +126,7 @@ runs:
         path: |-
           ${{ steps.build.outputs.gh && fromJson(steps.build.outputs.gh).dest }}
           ${{ steps.build.outputs.w3c && fromJson(steps.build.outputs.w3c).dest }}
-        name: spec-prod-result
+        name: ${{ inputs.ARTIFACT_NAME }}
```

Comment:
I think we might also need `overwrite: true`?

Reply:
I feel like the current behavior (a hard error) is better, so you know you have something to fix and we don't overwrite unexpectedly? I don't feel strongly though.

Reply:
Oh good point. I wasn't not sure if it'll work without that on subsequent job runs, but reading the docs, I believe it'll work.
A hard error is definitely more suitable here (that user forgot to use a different artifact name for different spec).